api/cache/system.go

```diff
@@ -56,3 +56,8 @@ func (o *SystemCache) Put(key string, obj *object.Object) error {
 func (o *SystemCache) Delete(key string) bool {
 	return o.cache.Remove(key)
 }
+
+// SystemObjectKey is key to use in SystemCache.
+func SystemObjectKey(bucket, obj string) string {
```

Comment:
Why this replaced method from `BucketInfo`?
```
// SystemObjectKey is key to use in SystemCache.
func (b *BucketInfo) SystemObjectKey(obj string) string {
	return b.Name + obj
}
```

Reply:
As I see we use SystemObjectKey only in the context of cache and don't think we will use in other cases. Maybe i'd be better to keep structs related to cache in cache?

Reply:
Moved it into `layer/system_object.go`.